DEVELOPERS.md

```diff
@@ -73,3 +73,53 @@ Starting with version 4.0, Bash is licenced under GPLv3. Because of this, macOS
 ```bash
 brew install bash
 ```
+
+
+## Running tests against Databricks
+
+The test suite for Databricks/Spark backend by default runs tests against
+a local spark db in a container, for reliablility and speed.
+
+Open source Spark and Databricks' Spark are very similar, and this provides
+a good enough tests for the SQL parts of a backend.
+
+However, we still need to run tests against an actual Databricks instance, as
```

Comment:
"...which given the limitations of Databricks CE will have to be a hosted SaaS edition..." ?

Reply:
I don't understand this. Databricks is a SaaS, regardless of CE or paid-for? We're definitely not going to try self-host the enterprise SaaS like NHSD do (given they are trying to avoid that!)

How about:
```
However, we still need to run tests against an actual Databricks instance, which is only available in SaaS form. We use the free Community Edition version to do so, but it is limited, slow, and not 100% reliable, so we do not run it by default, it needs to be manually run.
```

Reply:
I've updated this section with something along the line above, let me know if it makes more sense.